src/registry_instance.jl

```diff
@@ -14,10 +14,21 @@ function RegistryInstance(path::AbstractString)
 
     for (uuid, info) in d["packages"]
         uuid = UUID(uuid)
-        info
         name = info["name"]
         pkgpath = info["path"]
-        pkg = PkgEntry(pkgpath, path, name, uuid)
+
+        p_repo = ""
+        try
+            p = parsefile(joinpath(path, pkgpath, "Package.toml"))
+            p_name = p["name"]
+            p_uuid = UUID(p["uuid"])
+            p_repo = p["repo"]
+            @assert p_name == name
```

Comment:
Not sure I follow the need for assertions here and the try/catch?

Reply:
The try catch is necessary due to the tests. The `Foobar` registry contains `DownDep` which doesn't actually exist in file/folder form so when trying to read the `Package.toml` file it doesn't exist. 

The asserts are just to make sure the data in Package.toml matches what we see in the Regitry.toml. Unsure if there is a better way to confirm that than to just check for equality.

Reply:
- Can we not modify the tests to include this file and bin the `try/catch`?
- I think it's safe to assume that registry files will match each other as this is all an automated process when registering (or at least should be).

Reply:
sure makes sense to me.